niceml/data/datasets/dfdataset.py

```diff
@@ -27,51 +37,148 @@ class RegDataInfo(DataInfo):
     data: dict
 
     def get_info_dict(self) -> dict:
+        """
+        The get_info_dict function returns a dictionary containing the dataid
+        and all the data in self.data.
+
+        Returns:
+            A dictionary containing the dataid and all the other key-value pairs in self
+        """
         info_dict = dict(dataid=self.dataid)
         info_dict.update(self.data)
         return info_dict
 
     def get_identifier(self) -> str:
+        """
+        The get_identifier function returns the dataid of this object.
+
+        Returns:
+            The dataid
+        """
         return self.dataid
 
+    def __getattr__(self, item) -> Any:
+        """
+        The __getattr__ function is called when an attribute lookup has not found the attribute
+        in the usual places (i.e. it is not an instance attribute nor is it
+        found in the class tree for self).
+
+        Args:
+            item: Access the value of a key in the dictionary
+
+        Returns:
+            The value of the key in the data dictionary
+        """
+        return self.data[item]
+
 
 class DfDataset(Dataset, Sequence):  # pylint: disable=too-many-instance-attributes
     """Dataset for dataframes"""
 
-    def __init__(  # pylint: disable=too-many-arguments
+    def __init__(  # ruff: noqa: PLR0913
         self,
         id_key: str,
         batch_size: int,
         set_name: str,
-        df_loader: DfLoader,
-        df_path: str,
+        data_location: Union[dict, LocationConfig],
+        df_path: str = "{set_name}.parq",
```

Comment:
Gibt es hier eine Variable für?

Reply:
Eine Variable wofür genau?

Reply:
für df_path. Vllt. gibt es eine passende in expfilenames.py als default Wert.

Reply:
Ja, das ist eine gute Idee 👍 
Ich schaue oder ergänze eine entsprechende Variable.

Reply:
Es gibt nun die Variable `ExperimentFilenames.SUBSET_NAME`, die als Standardwert gesetzt ist.